lib/types/enums.ts

```diff
@@ -47,6 +47,9 @@ export enum ReputationEvent {
   PacketTimeout = 2,
   SwapFailure = 3,
   SwapSuccess = 4,
+  UnparsableMessage = 5,
```

Comment:
Missing an 'e' in "Unparseable".

Reply:
Thanks.

Reply:
Interesting I did not know that, but yeah better to stick to the spelling with the e since that's what we're using elsewhere.